<launch>
  
  <include file="$(find pid)/launch/pid_erp42_control.launch">
  </include>
  
  <node pkg="erp42_serial" type="pid_erp42.py" name="pid_erp42"/>
  
  <!--group ns="Serial_communicate"-->
  <node pkg="erp42_serial" type="erp42_com.py" name="erp42_com"/>
  <!--/group-->
  
</launch>
